src/ad-hoc-visualizations/needs-review/get-notification-message-for-needs-review.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ScanIncompleteWarningId } from 'common/types/scan-incomplete-warnings';
+import { isEmpty } from 'lodash';
+import { DictionaryStringTo } from 'types/common-types';
+
+export const getNotificationMessage = (
+    selectorMap: DictionaryStringTo<any>,
+    warnings: ScanIncompleteWarningId[],
+) => {
+    if (isEmpty(selectorMap) && isEmpty(warnings)) {
+        return 'Congratulations!\n\nNeeds review found no instances to review on this page.';
+    }
+
+    let baseMessage = 'No instances to review found.';
```

Comment:
This code would be even better if you extracted this assignment to a function in the same way you did getMessagePrefix. That way, the code would simply read: 

- getMessageBase
- getMessagePrefix
- return the combined value.

It's hard to get simpler than that. :)

Reply:
Agreed! Made the change.